### DISCUSSION TITLE
Update script

### DISCUSSION THREAD

**Discussion by olsson82** [olsson82/rivendellweb#31] [Ideas]
Im thinking about an built in update script for easy update the system thru the admin panel.

Still just an idea but it can be reality.

**Comment by alastairtech** [comment]
Something built in would be amazing. I made my own tui installer with an upgrade function here: https://github.com/alastairtech/rivendell-arm/blob/main/webinterface.sh

I've been testing on a couple of production stations and liking things so far. Love your work! 

Don't worry, these stations are backed up daily and almost unbreakable. 😂

**Reply by olsson82** [reply]
When the multitrack voicetracker is stable for a release am going to work on an update script.

It should be simple to update, not everyone likes to work in the terminal 😃 

Your script looks good to.

Always good with backups.
Maby that could be a feature in the future, automatic database backup 😁
